service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -73,8 +97,11 @@ public void shouldMakeCorrectCallForFees(Boolean toggleState) {
         givenToggle(toggleState);
 
         assertThat(featureToggleService.isFeesEnabled()).isEqualTo(toggleState);
+        featureToggleService.isCtscReportEnabled();
 
-        verify(ldClient).boolVariation(eq("FNP"), any(LDUser.class), eq(false));
+        verify(ldClient, atLeastOnce()).boolVariation(eq("FNP"), ldUser.capture(), eq(false));
```

Comment:
`atLeastOnce()` is unnecessary. There is only going to be one interaction with the "FNP" feature flag as `isCtscReportEnabled()` uses "CTSC"